pictureGenerator_HW/app.js

```diff
@@ -0,0 +1,20 @@
+const eventType= 'click';
+
+const randomButton = document.querySelector('#randomize');
+const widthImg = document.querySelector('.width');
+const heightImg = document.querySelector('.height');
+  
+
+
+
+
+const changeImgRandom = function(){
+
```

Comment:
to make this bonus work make make  a variable for both height and width and use them as part of the URL

similar to this 
```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```